CodeSniffer/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -74,6 +74,19 @@ protected function processReturn(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
         if ($isSpecialMethod === true) {
             return;
         }
+        
+        // Skip if function comment contains inheritDoc
+        for ($i=0; $i < 10; $i++) {
+            if (! isset($tokens[$commentStart+$i])) {
+                break;
+            }
+
+            $tokenContent = strtolower($tokens[$commentStart+$i]['content']);
+
+            if (strstr($tokenContent, '{@inheritdoc')) {
```

Comment:
According to docs (see https://www.phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag) this tag only inherits description.

In Symfony however it's used to inherit following:

* short description
* long description
* parameter list (the `@param` tag)
* return type (the `@return` tag)